tests/test_data_frame.py

```diff
@@ -0,0 +1,62 @@
+__author__ = 'admin'
+import json
```

Comment:
Not sure what this file is for, as there are no tests defined in it. Tests should be defined in functions with the prefix ```test_``` in their name, and the tests should assert something.